src/providers.ts

```diff
@@ -34,9 +34,21 @@ export const bitbucket: TemplateProvider = (input) => {
   }
 }
 
+export const sourcehut: TemplateProvider = (input) => {
+  const parsed = parseGitURI(input)
+  return {
+    name: parsed.repo.replace('/', '-'),
+    version: parsed.ref,
+    subdir: parsed.subdir,
+    url: `https://git.sr.ht/~${parsed.repo}/tree/${parsed.ref}/item${parsed.subdir}`,
+    tar: `https://git.sr.ht/~${parsed.repo}/superd/archive/${parsed.ref}.tar.gz`
```

Comment:
I am trying it locally https://git.sr.ht/~pi0/unjs-template/superd/archive/main.tar.gz gives 404. Maybe it has been changed or I'm doing something wrong?